plothist/plotters.py

```diff
@@ -368,8 +364,8 @@ def compare_two_hist(
         The main axes for the histogram comparison. If fig, ax_main and ax_comparison are None, a new axes will be created. Default is None.
     ax_comparison : matplotlib.axes.Axes or None, optional
         The axes for the comparison plot. If fig, ax_main and ax_comparison are None, a new axes will be created. Default is None.
-    ratio_uncertainty : str, optional
-        How to treat the uncertainties of the histograms when comparison = "ratio" ("uncorrelated" for simple comparison, "split" for scaling and split hist_1 and hist_2 uncertainties). Default is "uncorrelated"
+    **comparison_kwargs : optional
+        Arguments to be passed to plot_comparison().
```

Comment:
If they are not provided explicitly, the following arguments are passed by default: [...] (same trick as in `compare_data_mc()`)